.gitignore

```diff
@@ -3,3 +3,12 @@
 /vendor/
```

Comment:
this line can be moved down with the other ``vendor`` lines to make it easier to read in future